docs/connectors/kafka-connect-connector.mdx

```diff
@@ -0,0 +1,118 @@
+
+Let's begin.
+
+## How it works
+
+To use the Kafka Connect wrapper connector, you'll need to:
+
+1. Download the [`conduit-kafka-connect-wrapper`](https://github.com/ConduitIO/conduit-kafka-connect-wrapper) latest release,
+at the time of writing this, we'll go with [v0.4.3](https://github.com/ConduitIO/conduit-kafka-connect-wrapper/releases/tag/v0.4.3)
+and download the file `conduit-kafka-connect-wrapper-v0.4.3.zip`.
+1. Download Kafka Connect JARs and any dependencies you would like to add.
+1. Create a pipeline configuration file.
+1. Run Conduit.
+
+
+## Setup
+
+To begin, download the [`conduit-kafka-connect-wrapper`](https://github.com/ConduitIO/conduit-kafka-connect-wrapper) [latest release](https://github.com/ConduitIO/conduit-kafka-connect-wrapper/releases),
+at the time of writing this, we'll go with `v0.4.3` and download the file `conduit-kafka-connect-wrapper-v0.4.3.zip`.
+
+Downloading the release is our **preferred** option to get the connector JAR. However, another option to get the JAR file is
+to build it from source, this could be useful in case you added your own changes to the connector and wanted to test them
+out, to do that we will clone the connector:
+```
+git clone git@github.com:ConduitIO/conduit-kafka-connect-wrapper.git
+```
+
+Then, we need to build the connector. The Kafka Connect wrapper connector is written in Java, so it needs to be compiled.
+```bash
+cd conduit-kafka-connect-wrapper
+
+./scripts/dist.sh
+```
+
+Running `scripts/dist.sh` will create a directory called `dist` with following contents:
+1. A script (which runs the connector). This script starts a connector instance.
+1. The connector JAR itself.
+1. Directory `libs`. This is where you put the Kafka connector JARs and their dependencies (if any).
+
+Now that we have everything setup, we can add the Kafka Connect connectors.
+
+## Download a connector and its dependencies
+
+The `libs` directory is where you put the Kafka Connect connector JARs and their dependencies (if any). The wrapper 
+plugin will automatically load connectors and all the other dependencies from a `libs` directory.
+
+To download a connector from a Maven repository and all of its dependencies, you can use `scripts/download-connector.sh`.
+For example:
+```shell
+./scripts/download-connector.sh io.example jdbc-connector 2.1.3
+```
+For usage, run `./scripts/download-connector.sh --help`.
+
+You can also download them manually if needed, we will use the PostgreSQL Kafka Connect JDBC Connector. To install, add the following:
+- [Aiven's Kafka Connect JDBC Connectors](https://github.com/aiven/jdbc-connector-for-apache-kafka)
+- [Postgres Connector JAR](https://repo1.maven.org/maven2/org/postgresql/postgresql/42.3.3/postgresql-42.3.3.jar)
+
+This connector allows you to connect to any [JDBC database](https://en.wikipedia.org/wiki/Java_Database_Connectivity#:~:text=Java%20Database%20Connectivity%20(JDBC)%20is,Edition%20platform%2C%20from%20Oracle%20Corporation.).
```

Comment:
Maybe simplify the link to https://en.wikipedia.org/wiki/Java_Database_Connectivity (without the highlighted text)?